src/include/containers/BlockingList.hpp

```diff
@@ -80,7 +80,7 @@ class BlockingList : public List<T>
 		List<T>::clear();
 	}
 
-	LockGuard getLockGuard() { return LockGuard{_mutex}; }
+	pthread_mutex_t &mutex() { return _mutex; }
```

Comment:
Where is this used? Is it required?

Reply:
Yes, check the other changes.

Reply:
Oh now I get it, stupid me :man_facepalming:.